charts/fluentd-elasticsearch/templates/_helpers.tpl

```diff
@@ -40,6 +40,9 @@ helm.sh/chart: {{ include "fluentd-elasticsearch.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- with .Values.labels }}
+{{ toYaml . }}
```

Comment:
```suggestion
{{- toYaml . | nindent 8 }}
```

Reply:
@monotek the change you suggest fails yaml parsing. Since it goes into the fluentd-elasticsearch.labels template and is in each template file fed through `nindent 4` it doesn't come out as valid yaml.

Reply:
@monotek what do you think?